apps/judicial-system/web/src/routes/Prosecutor/InvestigationRequest/Defendant/DefendantForm.tsx

```diff
@@ -72,6 +62,12 @@ const DefendantForm: React.FC<Props> = (props) => {
     accusedAddress: {
       validations: ['empty'],
     },
+    defenderEmail: {
+      validations: ['email-format'],
+    },
+    defenderPhoneNumber: {
+      validations: ['phonenumber'],
+    },
```

Comment:
Shouldn't email and phone number validation be added to the hearing arrangement screens in the court flow?

Reply:
Thanks for spotting this. In #5472 we stopped using these validations in favour of [these ones](https://github.com/island-is/island.is/blob/main/apps/judicial-system/web/src/utils/validate.ts#L68) but we forgot to remove them. I've created a task for that [here](https://app.asana.com/0/1199153462262248/1201465012065696/f). 

I think that should be done in a separate PR.